README.md

```diff
@@ -119,7 +119,22 @@ Build the source and use js from build folder to your project
 $ npm run build
 ```
 
-
+## How to test a preset pull request
+
+1. Build the preset: `npm run build` (done by author)
+2. [Create a PR](https://github.com/mautic/grapesjs-preset-mautic/pulls) (done by author)
+3. Fork the repo: `gh repo fork mautic/grapesjs-preset-mautic`
+4. Clone the repo: `gh repo checkout USERNAME/grapesjs-preset-mautic && gh pr checkout <pr number>`
+5. Change into the plugin directory: e.g. `cd mautic/plugins/GrapesJSBuilderBundle`
+6. Change the code for the import path of the preset in `plugins/GrapesJsBuilderBundle/Assets/library/js/builder.service.js` to the local version of the preset. E.g.
+  ```js
+  // import grapesjsmautic from 'grapesjs-preset-mautic';
+  import grapesjsmautic from '../../../../../../grapesjs-preset-mautic/src';
```

Comment:
```suggestion
import grapesjsmautic from '../../../../../../grapesjs-preset-mautic/src';
import mjmlService from '../../../../../../grapesjs-preset-mautic/dist/mjml/mjml.service';
import contentService from '../../../../../../grapesjs-preset-mautic/dist/content.service';```

Reply:
Please add the missing code block closure after the first block - I can't in a comment as that ends the comment!